sbi/inference/snpe/snpe_base.py

```diff
@@ -93,7 +93,7 @@ def __call__(
         num_rounds: int,
         num_simulations_per_round: OneOrMore[int],
         x_o: Optional[Tensor] = None,
-        external_data: Optional[Tensor] = None,
+        external_data: Optional[Tuple[Tensor, Tensor]] = None,
```

Comment:
A workable default would be a tuple of emtpy tensors, wouldn't it? Then no more specific case-handling at prepend time.

Reply:
`external... --> presimulated`? Another idea for names